aten/src/ATen/native/sparse/mps/SparseMPSTensorMath.mm

```diff
@@ -888,5 +892,114 @@ static void sparse_mask_intersection_out_mps_kernel(
       /*coalesce_mask=*/false);
 }
 
+Tensor sparse_sparse_matmul_mps(const Tensor& mat1_, const Tensor& mat2_) {
+  TORCH_CHECK(mat1_.is_sparse() && mat2_.is_sparse(),
```

Comment:
A lot of these should probably be TORCH_CHECK_VALUE, if the other dispatches use this, we should probably update them all at once though.